server/cluster/manager_test.go

```diff
@@ -190,6 +191,12 @@ func testDropTable(ctx context.Context, re *require.Assertions, manager Manager,
 }
 
 func testAllocTableIDWithMultiThread(ctx context.Context, re *require.Assertions, manager Manager, clusterName string, tableID uint64) {
-	go testAllocTableID(ctx, re, manager, node1, clusterName, defaultSchema, table1, tableID)
+	wg := sync.WaitGroup{}
+	wg.Add(1)
+	go func() {
+		testAllocTableID(ctx, re, manager, node1, clusterName, defaultSchema, table1, tableID)
+		wg.Done()
+	}()
 	testAllocTableID(ctx, re, manager, node2, clusterName, defaultSchema, table1, tableID)
```

Comment:
Only two goroutines are not enough for multi-thread testing. How about starting a dozen of goroutines to do the allocating table id test?